packages/@aws-cdk/pipelines/lib/pipeline.ts

```diff
@@ -145,6 +153,27 @@ export interface CdkPipelineProps {
   readonly supportDockerAssets?: boolean;
 }
 
+/**
+ * Optional settings to configure the asset publishing step
+ */
+export interface AssetPublishingOptions {
```

Comment:
Not a big fan of this. I think I'd rather keep the options object flat.

Reply:
I can add it to the top level object. I was just wondering as we now have at least two settings if we want to group them together as asset stuff.

But if you do not see the need for this I can change it.

Reply:
We have it as a design guideline that we put as much as we can into top-level properties. It's a tough balancing act, but for 2 fields I don't see the need yet.

Reply:
Done